node/forest_libp2p/bitswap/src/proto/bitswap_pb.rs

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019-2023 ChainSafe Systems
```

Comment:
There is in-progress work in `rust-libp2p` to embed all generated proto files so that it won't depend on protoc anymore